service/docs/source/geopm_pio.7.rst

```diff
@@ -0,0 +1,200 @@
+
+geopm_pio(7) -- GEOPM PlatformIO interface
+==========================================
+
+DESCRIPTION
+-----------
+
+The ``PlatformIO`` class provides a high-level interface for signals
+(system monitors) and controls (system settings).  There are a large
+number of built-in signals and controls.  These built-in signals and
+controls include hardware metrics, hardware settings, and signals
+derived from application behavior.  Application behavior is tracked by
+GEOPM's integration with MPI and OpenMP and also by application use of
+the `geopm_prof_c(3) <geopm_prof_c.3.html>`_ mark-up interface. In
+addition to the built-in features, ``PlatformIO`` can be extended
+through the `geopm::IOGroup(3) <GEOPM_CXX_MAN_IOGroup.3.html>`_ plugin
+interface to provide arbitrary signals and controls.
+
+A domain is a discrete component within a compute node where a signal
+or control is applicable.  For more information about the
+``geopm_domain_e`` enum and the hierarchical platform description see
+`geopm::PlatformTopo(3) <GEOPM_CXX_MAN_PlatformTopo.3.html>`_.  A
```

Comment:
Some of the readers of man 7 geopm_pio may be end-users trying to understand which signals/controls they are interested in. Should this also mention `geopmread --domain`?

Reply:
Added a suggestion to see geopmread/write for CLI-based exploration of PlatformIO